Makefile

```diff
@@ -10,15 +10,29 @@ start-app:
 
 init:
 	make start-wp
-	@echo "Waiting for 15s for containers to be ready..."
-	# Change 15 to a higher number if it fails to install
-	@sleep 15
+	@echo "Waiting for 30s for containers to be ready..."
+	# Change 30 to a higher number if it fails to install
+	@sleep 30
```

Comment:
Can we somehow set this as a variable in one of our environment files so that it's adjustable locally and doesn't get accidentally committed?

Reply:
Definitely

Reply:
I've opted to load the value from a local shell script that exports the variable - that seemed like the easiest way given that we are in a Make/Shell environment. I've included 30s as a default value since that seems to work for me.